Modules/MessageView/Presenter/MessageViewPresenter.swift

```diff
@@ -158,8 +163,13 @@ extension MessageViewPresenter {
     public func show(
         contentView: UIView,
         position: MessageViewModel.Position = .bottom,
+        messageViewBackgroundColor: UIColor = .white,
```

Comment:
To reduce the parameters and keep the flexibility you could pass a appearance configuration containing parameters like background color position and others. This can be created for animations as well.